styleguide.config.js

```diff
@@ -18,17 +18,96 @@ module.exports = {
     require: [
         path.join(__dirname, './src/global.sass'),
     ],
+	sections: [
+		{
+			name: 'Getting started with Add-ons',
+			content: 'README.md',
+			href: 'https://build.localbyflywheel.com/project/',
+			external: true,
+		},
+		{
+			name: 'Components',
+			sections: [
+				{
+					name: 'Alerts',
+					components: 'src/components/alerts/**/[A-Z]*.tsx',
+					ignore: '**/_private/*.tsx',
+				},
+				{
+					name: 'Buttons',
+					components: 'src/components/buttons/**/[A-Z]*.tsx',
+					ignore: '**/_private/**/*.tsx',
+				},
+				{
+					name: 'Inputs',
+					components: 'src/components/inputs/**/[A-Z]*.tsx',
+					ignore: '**/_private/*.tsx',
+				},
+				{
+					name: 'Layout',
+					components: 'src/components/loaders/**/[A-Z]*.tsx',
```

Comment:
`Layout` and `Loaders` have the same value for `components`. Is this intentional?

Reply:
Absolutely not. Good catch!